include/net/lwm2m.h

```diff
@@ -25,6 +25,16 @@
 #define IPSO_OBJECT_TEMP_SENSOR_ID			3303
 #define IPSO_OBJECT_LIGHT_CONTROL_ID			3311
 
+/**
+ * LwM2M context structure
+ *
+ * @details Context structure for the LwM2M high-level API.
+ */
+struct lwm2m_ctx {
+	/** Net app context structure */
```

Comment:
Woops.. by the end of the patch set this is a net_app_ctx.. I probably didn't adjust the comment mid-patch series.  I'll fix.